application/views/dashboard/index.php

```diff
@@ -82,11 +82,14 @@ function echoQrbCalcLink($mygrid, $grid, $vucc) {
 <?php } ?>
 </div>
 
-<!-- Map -->
-<div id="map" style="width: 100%; height: 350px"></div>
 
 <div style="padding-top: 0px; margin-top: 5px;" class="container dashboard">
 
+<?php if((!$this->config->item('map_at_right')) && (!$this->config->item('hide_map'))) { ?>
+<!-- Map -->
+<div id="map" style="width: 100%; height: 350px"></div>
```

Comment:
We should add a little margin at the bottom to separate from other content.
```
  style="width: 100%; height: 350px; margin-bottom: 15px;"
```

Reply:
> We should add a little margin at the bottom to separate from other content.
> 
> ```
>   style="width: 100%; height: 350px; margin-bottom: 15px;"
> ```

Thank you!